static/app/views/explore/spans/quotaAlert.tsx

```diff
@@ -0,0 +1,146 @@
+import styled from '@emotion/styled';
+
+import {Alert} from 'sentry/components/core/alert';
+import Link from 'sentry/components/links/link';
+import {normalizeDateTimeParams} from 'sentry/components/organizations/pageFilters/parse';
+import {t, tct} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {parsePeriodToHours} from 'sentry/utils/duration/parsePeriodToHours';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+import usePageFilters from 'sentry/utils/usePageFilters';
+import {usePerformanceSubscriptionDetails} from 'sentry/views/performance/newTraceDetails/traceTypeWarnings/usePerformanceSubscriptionDetails';
+import {usePerformanceUsageStats} from 'sentry/views/performance/newTraceDetails/traceTypeWarnings/usePerformanceUsageStats';
+
+function dateFormatter(date: Date): string {
+  return date.toLocaleDateString('en-US', {
+    month: 'short',
+    day: '2-digit',
+    year: 'numeric',
+  });
+}
```

Comment:
Looks like there's a `getFormattedDate` helper, can we use that here to define the date format?